CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md

```diff
@@ -59,6 +87,13 @@ Hints:
 -   Filter in the Configuration Pane by adding searches of the form
     \<column_name> =="\<search text>" E.g. first_name=="Nick".
 
+Full Example:
+
+<div style="text-align: center">
+    <img src="../ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/resources/ImportDatabaseFullExample.png" alt="Database Importer Connection
+    example with all details" />
+</div>
+<br/>
 ## Add and Remove Connections to Import Queries From
```

Comment:
Feels like the detailed steps above should actually be added here?

Reply:
> The following is a high level workflow an analyst can follow (explained
> in detail later on):

This made me think we should move the newly added connection details to the bottom detailed section. But not a big deal.